plugins/org.orbisgis.groovyeditor/src/org/orbisgis/ui/editors/groovy/GroovyJob.java

```diff
@@ -90,7 +112,7 @@ protected IStatus run(IProgressMonitor iProgressMonitor) {
             message = result.toString();
         }
         if(status == IStatus.OK) {
-            LOGGER.info(message);
+            //LOGGER.info(message);
```

Comment:
Commented ?

Reply:
I can delete this condition because there is already the "Groovy script successfully executed." message that displays in the groovy output.